TRB_label_assist/clustering_examples.ipynb

```diff
@@ -26,12 +26,13 @@
                 "%autoreload 2\n",
                 "\n",
                 "from uuid import UUID\n",
-                "\n",
+                "import itertools\n",
                 "import emission.storage.timeseries.abstract_timeseries as esta\n",
                 "import emission.storage.decorations.trip_queries as esdtq\n",
                 "import emission.core.get_database as edb\n",
-                "\n",
-                "import mapping"
+                "import emission.analysis.modelling.trip_model.run_model as eamtr\n",
+                "import mapping\n",
+                "import emission.analysis.modelling as eam"
```

Comment:
what do we use this for? we typically import the leaf modules and not a package

Reply:
we use `find_plot_clusters` from this file. That's our custom .py file. 

We have used mapping and clustering in some other files as well:

```
import mapping
import clustering
```

Reply:
1. which other files?
2. My comment was primarily around `import emission.analysis.modelling as eam` (where `emission.analysis.modelling` is the package) elsewhere. We typically import individual modules.

Reply:
1. My understanding was around `mapping.py` and `clustering.py` which are leaf modules.
 
2.`eam` is unnecessary here. We are importing anyways importing `eamtr` in case when we need to run the model. I'll remove it.